fieldservice_agreement/models/agreement.py

```diff
@@ -15,6 +15,10 @@ class Agreement(models.Model):
                                      compute='_compute_equipment_count')
     fsm_location_id = fields.Many2one('fsm.location', string="FSM Location")
 
+    serviceprofile_ids = fields.Many2many('agreement.serviceprofile',
+                                          default='_set_service_ids',
+                                          string="Service Profiles")
```

Comment:
@osi-scampbell Should be moved to `agreement_serviceprofile` if not already there.